src/TorchSharp/Utils/tensorboard/GifEncoder/NeuQuant.cs

```diff
@@ -0,0 +1,487 @@
+using System;
```

Comment:
Copyright header, please.

Reply:
The code I copied from here, how can I write.
https://github.com/avianbc/NGif/blob/master/Components/NeuQuant.cs